From e8004dad869568692ca2a45c04e8464ce48cc4d7 Mon Sep 17 00:00:00 2001
From: Ajay Kumar Gupta <ajay.gupta@ti.com>
Date: Mon, 26 Mar 2012 17:55:12 +0530
Subject: [PATCH 193/193] usb: musb: fix wifi failure on am335x EVM

Fixes the DWA-125 wifi device issue seen when wifi interface is enabled using
"ifconfig ra0 up" showing below memory alloc error.

[  138.424302] <-- ERROR in Alloc TX TxContext[1] HTTX_BUFFER !!
[  138.430459] <-- RTMPAllocTxRxRingMemory, Status=3 [  138.435376] ERROR!!!
RTMPAllocDMAMemory failed, Status[=0x00000003] [  138.441913]
!!! rt28xx Initialized fail !!!

Fixed by increasing the CONSISTENT DMA size level to 5 from 4.
-CONFIG_FB_DA8XX_CONSISTENT_DMA_SIZE=4
+CONFIG_FB_DA8XX_CONSISTENT_DMA_SIZE=5
---
 arch/arm/configs/am335x_evm_defconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/configs/am335x_evm_defconfig b/arch/arm/configs/am335x_evm_defconfig
index a9350a4..56d8b10 100644
--- a/arch/arm/configs/am335x_evm_defconfig
+++ b/arch/arm/configs/am335x_evm_defconfig
@@ -1775,7 +1775,7 @@ CONFIG_FB_CFB_IMAGEBLIT=y
 # CONFIG_FB_SMSCUFX is not set
 # CONFIG_FB_UDL is not set
 CONFIG_FB_DA8XX=y
-CONFIG_FB_DA8XX_CONSISTENT_DMA_SIZE=4
+CONFIG_FB_DA8XX_CONSISTENT_DMA_SIZE=5
 # CONFIG_FB_VIRTUAL is not set
 # CONFIG_FB_METRONOME is not set
 # CONFIG_FB_BROADSHEET is not set
-- 
1.7.9.5

